### DISCUSSION TITLE
Сведения о транзакции Переведите 2,93 ETC

### DISCUSSION THREAD

**Discussion by vysochansckij** [blockscout/blockscout#11414] [Support]
Здравствуйте. Помогите пожалуйста при переводе со своего кошелька на свой другой кошелёк указал не неправильную сеть помогите вернуть данную транзакцию обратно Сведения о транзакции
Переведите 2,93 ETC хеш 0x9e47521d028777bbd7bfdef482f22c54c50d2bbc053181f8a63268500cb946bd

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
